client/my-sites/current-site/notice.jsx

```diff
@@ -142,51 +131,21 @@ export class SiteNotice extends Component {
 	}
 }
 
-export default connect(
-	( state, ownProps ) => {
-		const siteId = ownProps.site && ownProps.site.ID ? ownProps.site.ID : null;
-		const sectionName = getSectionName( state );
-		const isMigrationInProgress =
-			isSiteMigrationInProgress( state, siteId ) || isSiteMigrationActiveRoute( state );
-
-		// Avoid passing `messagePath` as a prop to `SiteNotice` as it changes frequently and
-		// thus will cause a lot of re-renders.
-		const messagePath = `calypso:${ sectionName }:sidebar_notice`;
-
-		return {
-			isDomainOnly: isDomainOnlySite( state, siteId ),
-			isEligibleForFreeToPaidUpsell: isEligibleForFreeToPaidUpsell( state, siteId ),
-			isJetpack: isJetpackSite( state, siteId ),
-			activeDiscount: getActiveDiscount( state ),
-			hasDomainCredit: hasDomainCredit( state, siteId ),
-			canManageOptions: canCurrentUser( state, siteId, 'manage_options' ),
-			productsList: getProductsList( state ),
-			domains: getDomainsBySiteId( state, siteId ),
-			isPlanOwner: isCurrentUserCurrentPlanOwner( state, siteId ),
-			currencyCode: getCurrentUserCurrencyCode( state ),
-			domainUpsellNudgeDismissedDate: getPreference( state, DOMAIN_UPSELL_NUDGE_DISMISS_KEY ),
-			isSiteWPForTeams: isSiteWPForTeams( state, siteId ),
-			isMigrationInProgress,
-			hasJITM: getTopJITM( state, messagePath ),
-		};
-	},
-	( dispatch ) => {
-		return {
-			clickClaimDomainNotice: () =>
-				dispatch(
-					recordTracksEvent( 'calypso_domain_credit_reminder_click', {
-						cta_name: 'current_site_domain_notice',
-					} )
-				),
-			clickDomainUpsellGo: () =>
-				dispatch(
-					recordTracksEvent( 'calypso_upgrade_nudge_cta_click', {
-						cta_name: 'domain-upsell-nudge',
-					} )
-				),
-			clickDomainUpsellDismiss: () => {
-				dispatch( savePreference( DOMAIN_UPSELL_NUDGE_DISMISS_KEY, new Date().toISOString() ) );
-			},
-		};
-	}
-)( localize( SiteNotice ) );
+export default connect( ( state, ownProps ) => {
+	const siteId = ownProps.site && ownProps.site.ID ? ownProps.site.ID : null;
+	const isMigrationInProgress =
+		isSiteMigrationInProgress( state, siteId ) || isSiteMigrationActiveRoute( state );
+
+	return {
+		isDomainOnly: isDomainOnlySite( state, siteId ),
+		isEligibleForFreeToPaidUpsell: isEligibleForFreeToPaidUpsell( state, siteId ),
+		isJetpack: isJetpackSite( state, siteId ),
+		activeDiscount: getActiveDiscount( state ),
+		canManageOptions: canCurrentUser( state, siteId, 'manage_options' ),
+		productsList: getProductsList( state ),
+		domains: getDomainsBySiteId( state, siteId ),
+		currencyCode: getCurrentUserCurrencyCode( state ),
```

Comment:
Seems that `canManageOptions`, `productsList`, `domains`, and `currencyCode` are no longer used too.

Reply:
By removing `productsList` I think we can remove `<QueryProductsList />` as well.